mkosi/resources/mkosi.md

```diff
@@ -1280,11 +1284,11 @@ are mounted into the current working directory before running the script
 and `$SRCDIR` is set to point to the current working directory. The
 following scripts are supported:
 
-* If **`mkosi.prepare`** (`PrepareScript=`) exists, it is first called
+* If any **`mkosi.prepare`** (`PrepareScripts=`) exist, they are first called
```

Comment:
Documentation here should probably be clearer on how to use this. In a given directory there can only be a single file `mkosi.prepare`. So as a user I would wonder does this mean I can only have a single prepare script per preset or per `mkosi.conf.d` or is `mkosi.prepare` a directory and scripts will be picked up there?

Reply:
I see how this can be confusing with regards to file vs directory. I could revert to the singular and add a paragraph explaining how multiple scripts can be called at each point with `mkosi.conf.d`, `mkosi.preset` and/or the settings in `mkosi.conf`.

Is the following weird when talking about a single script but mentioning the setting as a plural?

```suggestion
* If **`mkosi.prepare`** (`PrepareScripts=`) exists, it is first called
```

Reply:
Losing the "any" improves readability. I'd prefer some `mkosi.prepare.d` support (well, for the others to), but than can wait for later.